client/my-sites/domains/domain-management/contacts-privacy/card.jsx

```diff
@@ -5,55 +5,136 @@
+
+		if ( privateDomain ) {
+			this.props.disableDomainPrivacy( selectedSite.ID, name );
+		} else {
+			this.props.enableDomainPrivacy( selectedSite.ID, name );
+		}
+	};
+
+	toggleContactInfo = () => {
+		const { selectedSite, contactInfoDisclosed, selectedDomainName: name } = this.props;
+
+		if ( contactInfoDisclosed ) {
+			this.props.redactDomainContactInfo( selectedSite.ID, name );
+		} else {
+			this.props.discloseDomainContactInfo( selectedSite.ID, name );
+		}
+	};
+
+	getPrivacyProtection() {
+		const { privateDomain, privacyAvailable } = this.props;
+		const { translate, isUpdatingPrivacy } = this.props;
+
+		if ( ! privacyAvailable ) {
+			return false;
+		}
+
+		return (
+			<Property label={ translate( 'Privacy Protection' ) }>
 				{
-					components: {
-						strong: <strong />,
-					},
+					<FormToggle
+						wrapperClassName="edit__privacy-protection-toggle"
+						checked={ privateDomain }
+						toggling={ isUpdatingPrivacy }
+						disabled={ isUpdatingPrivacy }
+						onChange={ this.togglePrivacy }
+					/>
 				}
-			);
+			</Property>
+		);
+	}
+
+	getContactInfoDisclosed() {
+		const {
+			translate,
+			isUpdatingPrivacy,
+			privateDomain,
+			privacyAvailable,
+			contactInfoDisclosureAvailable,
+			contactInfoDisclosed,
+			isPendingIcannVerification,
+		} = this.props;
+
+		if ( ! privacyAvailable || ! contactInfoDisclosureAvailable || privateDomain ) {
+			return false;
 		}
 
+		const contactVerificationNotice = isPendingIcannVerification ? (
+			<div class="edit__disclose-contact-information-warning">
+				<Gridicon icon="info-outline" size={ 18 } />
+				<p>
+					{ translate(
+						'You need to verify the contact information for the domain before you can disclose it publicly.'
+					) }
+				</p>
+			</div>
+		) : null;
+
+		return (
+			<div>
+				<Property label={ translate( 'Display my contact information in public WHOIS' ) }>
+					<FormToggle
+						wrapperClassName="edit__disclose-contact-information"
+						checked={ contactInfoDisclosed }
+						toggling={ isUpdatingPrivacy }
+						disabled={ isUpdatingPrivacy || isPendingIcannVerification }
+						onChange={ this.toggleContactInfo }
+					/>
+				</Property>
+				{ contactVerificationNotice }
+			</div>
+		);
+	}
+
+	render() {
+		const { translate, selectedDomainName } = this.props;
+
 		return (
 			<div>
 				<SectionHeader label={ translate( 'Domain Contacts' ) } />
 
 				<CompactCard className="contacts-privacy__card">
-					<p>{ privacyText }</p>
-					<ContactDisplay
-						selectedDomainName={ selectedDomainName }
-						privateDomain={ privateDomain }
-					/>
+					{ this.getPrivacyProtection() }
+
+					{ this.getContactInfoDisclosed() }
+
+					<ContactDisplay selectedDomainName={ selectedDomainName } />
+
 					<p className="contacts-privacy__settings-explanation">
 						{ translate(
 							'Domain owners are required to provide correct contact information. ' +
```

Comment:
Update it how?

Reply:
The thing I would update is actually the part after the fold: 
> {{a}}Learn more{{/a}} about private registration and GDPR protection.

I would think it might be clearer to have this be a bit more generic, e.g.
> {{a}}Learn more{{/a}} about domain privacy and how your contact information is handled.

Or even use "privacy protection" to match the name of the control:
> {{a}}Learn more{{/a}} about privacy protection and how your contact information is handled.

Regardless, this may need input from Wendy and/or be deferred.